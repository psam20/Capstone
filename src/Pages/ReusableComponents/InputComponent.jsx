import React from 'react';
import TextField from '@material-ui/core/TextField';


const TextInput = (props)=>{

    return (
        <div className="form-group">
        <TextField required id="outlined-required" variant="outlined" label={props.name.toUpperCase()} name={props.name} type={props.type} defaultValue={props.value} onChange={props.onChange} fullWidth={true} multiline={props.multiline}/>
        </div>
    )
}

export default TextInput;