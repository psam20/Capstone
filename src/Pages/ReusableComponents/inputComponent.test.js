import React from 'react';
import {shallow } from 'enzyme';
import InputComponent from './InputComponent';
import { jssPreset } from '@material-ui/core';
describe('Input Component Testing',()=>{
    const props={
        name:name,
        onChange:jest.fn(),
        label:"ProductName",
        defaultValue:"ProductName",
        multiLine:true
    }
    let wrapper = shallow(<InputComponent {...props}/>)
    it('It should render Input Componet',()=>{
        expect(wrapper).toMatchSnapshot();
    })
})