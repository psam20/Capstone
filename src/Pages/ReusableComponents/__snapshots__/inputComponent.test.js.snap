// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component Testing It should render Input Componet 1`] = `ShallowWrapper {}`;
